website/content/2020/schedule.11ty.js

```diff
@@ -55,7 +55,7 @@ function list (iterable, mapper, sep='', lastSep) {
   return `${entries.slice(0, entries.length - 1).join(sep)}${lastSep}${entries[entries.length-1]}`
 }
 
-module.exports = class Schedule {
+class Schedule {
```

Comment:
Does this work?

Reply:
No, it shouldn't have cleanly merged. This was a mistake.